src/layout/layout_tree/tree.rs

```diff
@@ -17,6 +19,21 @@ pub enum Direction {
     Left
 }
 
+#[derive(Clone, Debug)]
+pub enum TreeError {
+    /// A Node can nt be found in the tree with this Node Handle.
```

Comment:
Yes, thank you I will be sure to fix this. 

Part of the big deal I make however is that there are sometimes spell checking issues on your PRs that I don't find but rather my computer finds (e.g: the little underline telling me there's a misspelling). We all make spelling mistakes, and part of this PR process should be to mitigate that, but I was concerned after that happened a few times that you didn't have spell checking enabled. 

This is one of the those cases where the computer isn't smart enough to tell me I made a mistake (at least on my editor), and I appreciate this process.